src/nemo_evaluator/environments/gym.py

```
@@ -437,15 +439,57 @@ def _wait_for_health(self) -> None:
+            logger.warning("Gym process group %d survived SIGTERM; sending SIGKILL", pgid)
+            self._signal_group(pgid, signal.SIGKILL)
+            self._wait_for_group_exit(pgid, timeout=5.0)
+
         try:
-            self._process.send_signal(signal.SIGTERM)
-            self._process.wait(timeout=10)
-        except (subprocess.TimeoutExpired, OSError):
-            self._process.kill()
             self._process.wait(timeout=5)
+        except subprocess.TimeoutExpired:
+            pass
```

Ruff (0.15.21)

[warning] Use `contextlib.suppress(subprocess.TimeoutExpired)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(subprocess.TimeoutExpired): ...`

(SIM105)